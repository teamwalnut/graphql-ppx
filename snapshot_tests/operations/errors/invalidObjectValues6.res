/**
 * This should not throw as it's an empty list
 */
module MyQuery = %graphql(`
    query {
        invalidObjectValuesLst(
            embeddedList: []
        )
    }
`)
@ocaml.doc("
 * This should throw as it's not a list
 ")
module MyQuery2 = %graphql(`
    query {
        invalidObjectValuesLst(
            embeddedList: {field: ""}
        )
    }
`)
