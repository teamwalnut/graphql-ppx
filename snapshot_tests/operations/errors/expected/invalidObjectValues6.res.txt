
  [1;31mWe've found a bug for you![0m
  [36msnapshot_tests/operations/errors/invalidObjectValues6.res[0m:[2m17:27-37[0m

  15 [2mâ”‚[0m     query {
  16 [2mâ”‚[0m         invalidObjectValuesLst(
  [1;31m17[0m [2mâ”‚[0m             embeddedList: [1;31m{field: ""}[0m
  18 [2mâ”‚[0m         )
  19 [2mâ”‚[0m     }
  20 [2mâ”‚[0m `)

  Invalid argument. Expected "[InputObject]!" but received "InputObject!".

